addons/custom_modules/CfgVehicles.hpp

```diff
@@ -4,8 +4,9 @@ class CfgVehicles {
     #define ADD_CUSTOM_MODULE(id) \
         class DOUBLES(GVAR(module),id): EGVAR(modules,moduleBase) { \
             displayName = CSTRING(DisplayName); \
+            category = QGVAR(category); \
```

Comment:
Is it needed here? And was there an issue before this was added?

Reply:
Yes, needed. No.